code/modules/cooking/machinery/smartfridge.dm

```diff
@@ -30,6 +30,11 @@
 
 	// what icon overlay to use to show its contents - set to NULL if no contents.
 	var/contents_path = "-plant"
+	var/list/display_tiers = list(
+		"25" = "-1",
+		"50" = "-2"
+	)
```

Comment:
Just make `var/display_tiers = 3` and `var/display_tier_amt = 25`. You can get the icon state value by taking the floor() of `(shown_contents.len / display_tier_amt) + 1` and then clamping it to display_tiers. No need to use a list here.